### ISSUE TITLE
Have tests in GitHub assets

### ISSUE THREAD

**Issue by worldofpeace** [Ulauncher/Ulauncher#390]
We'd like to add Ulanucher to nixpkgs https://github.com/NixOS/nixpkgs/pull/57859.
I'd like to be able to run the tests within the source distribution, a common practice in most distros.
However they aren't included, can you please consider doing this?

Currently to run tests I'd have to use the regular GitHub generated tarball, but this lacks the preferences prebuilt. And currently in nixpkgs we lack a way to use yarn internally.

**Comment by gornostal** [comment]
If I understand your request correctly, source tarrball generated by default is not working for you because it doesn't have preferences pre-built, and a tarball generated by Ulauncher CI process is lacks the tests, right?

I'm not sure I want to include tests into the tarball because they are run when the release is generated.

**Comment by worldofpeace** [comment]
Yes we'd like the tests at https://github.com/Ulauncher/Ulauncher/tree/dev/tests in these assets https://github.com/Ulauncher/Ulauncher/releases/download/5.3.0-beta1/ulauncher_5.3.0.beta1.tar.gz

> I'm not sure I want to include tests into the tarball because they are run when the release is generated.

There is a strong precedent in distros to be able to run tests in packages.
This allows us to verify the dependencies packaged by our distribution are compatible with the package and  it installs successfully. (nixpkgs moves very quickly).
Off the top of my head distros like Debian and Gentoo have dedicated testing phases for this exact purpose.
